### PR TITLE
Fix: dark mode toggle + asset icons in dark mode

### PR DESCRIPTION
**Pull request by katspaugh**
Fixes the dark mode switch and default value in store + adds a light background for token icons in dark mode.

Screenshot 2022-08-19 at 10 16 25

### CODE REVIEW

File: src/hooks/useDarkMode.ts [hunk 1]
```diff
@@ -3,18 +3,28 @@ import { selectSettings } from '@/store/settingsSlice'
...
   useEffect(() => {
-    const systemPreference = window.matchMedia(mediaQuery)
-    setIsDarkMode(settings.theme.darkMode ?? systemPreference.matches)
+    setIsDarkMode(settings.theme.darkMode ?? isSystemDarkMode())
+  }, [settings.theme.darkMode])
+
+  return isDarkMode
+}
 
+export const useDarkModeTheme = () => {
```
Comment (iamacook) [inline src/hooks/useDarkMode.ts:22]: Nit: I think `useTheme` or something similar would be a bit better, although MUI has a `useTheme` hook already.

**Review by iamacook — APPROVED** [review]

File: src/hooks/useDarkMode.ts (hunk 1 above)
Comment (katspaugh) [reply src/hooks/useDarkMode.ts:22]: `useLightDarkTheme`?

Comment (iamacook) [reply src/hooks/useDarkMode.ts:22]: Sounds good to me!

**Comment by usame-algan** [comment]
For me its still darkMode: true by default is that intended? My OS setting is set to light mode

**Pull request merged**